### DISCUSSION TITLE
Tin 0.4

### DISCUSSION THREAD

**Discussion by mehr32** [moa-engine/old-MOA#28] [Announcements]
- Fixing the statistics logger (storing the number of searches without additional information)
- Updating the Persian translation
- Removing faulty search engines
- Solving the Firefox support issue in opensearch.xml
- Fixing the "source code" button issue in the footer
- Removing extra files (the .vscode directory)
- Sync with SearٓXNG changes
- Rewriting version structure and adding version to footer
- Fixing the next page button repetition bug
- Fixing the alignment bug of the top buttons in the right-to-left (RTL) template.

![tin](https://github.com/moa-engine/MOA/assets/86001599/fba493d9-e997-4741-9701-c3bc7ad50f28)

<hr /><em>This discussion was created from the release <a href='https://github.com/moa-engine/MOA/releases/tag/v0.4'>Tin 0.4</a>.</em>

**Comment by marek2001** [comment]
When will that release introduced into official server?

**Reply by mehr32** [reply]
> When will that release introduced into official server?

I'm sorry for the long delay in responding. We're experiencing issues with the main server, including the fact that it is a donated server and I don't have access to update it. I will follow up again, but due to MoA being in beta, some issues are unavoidable. I apologize once more :(

**Comment by thmkhtry** [comment]
And I also left the team.
I am really sorry for the noiselessness of this work.
I decided to stay away from Moa for a while.

Best wishes for the kind and loving community and team of MOA
Taha
